### PR TITLE
Rails 6 requires Ruby 2.4.1+

### PR DESCRIPTION
**Pull request by jeremy**
References #32028

Skipping over 2.4.0 to sidestep the `"symbol_from_string".to_sym.dup` bug.

/cc @mtsmfm @matthewd

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,28 +60,21 @@ env:
     - "GEM=ac:integration"
 
 rvm:
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - 2.4
```
Comment (matthewd) [inline .travis.yml]: @jeremy  this means `2.4.1`... is that what you meant?

Comment (jeremy) [reply .travis.yml]: Nope. That's disappointing.

Comment (jeremy) [reply .travis.yml]: Flipped back to major.minor.tiny @ 6b3acf151829ee16a946c0d77529d236f115234c 🙃

**Comment by matthewd** [comment]
Mostly just because requiring a Ruby that's under a year old makes life difficult for people using OS packages.

Admittedly, we discussed it back in November, and there was a fair time before the final release... might be worth revisiting when we get to beta.

**Comment by mperham** [comment]
Seems reasonable.  One data point: I noticed Ubuntu 18.04 ships Ruby 2.5.1.  💯

**Pull request merged**